### ISSUE TITLE
Invalid Node element to append Iframe

### ISSUE THREAD

**Issue by vishnuprasad95** [aws-samples/aws-lex-web-ui#193]
The chatbot works good. The only issue I have is it doesn't load sometimes with this message displayed by the library . "Chatbot failed to load error:invalid node element to append Iframe".
I cant seem to find a fix for this. Can you help me out with this?

**Comment by bobpskier** [comment]
I've not seen that error before. Based on your description its intermittent. Can you describe the steps you take to reproduce? Is it on browser refresh, new browser window,  initial load on different systems, etc? Also which browser and version or across multiple browsers? Finally did you customize the snippet of html provided that loads up the iframe or are you using it as is? Would be useful if you can point us at a url to test against.

**Comment by vishnuprasad95** [comment]
I am using angular 6 and running the script in my webapp to load the chatbot into the Iframe. The issue happens occasionally when the chatbot initially loads up into the Iframe. Prompting a hard refresh resolves the issue , but I don't really understand what causes it. I use Google chrome. I was not able to reproduce the issue and it works fine most of the time .I checked on the standalone bot during the issue and it seemed to be working fine. Any ideas on what would be causing this issue? I have used the Html code as is . Also something trivial,what would be best way to not show the Iframe after the user logs out?
